databuilder/transformer/regex_str_replace_transformer.py

```diff
@@ -28,7 +28,11 @@ def init(self, conf):
 
     def transform(self, record):
         # type: (Any) -> Any
-        val = getattr(record, self._attribute_name)
+
+        if isinstance(record, dict):
```

Comment:
what is the new line for?

Reply:
Previously, this transformer also supported normal python object, but not `Dict`. This line basically supports `Dict`.